.github/workflows/spell.yml

```diff
@@ -0,0 +1,46 @@
+---
+# Copyright 2022 Samsung Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Documentation CI workflow
+# yamllint disable-line rule:truthy
+on:
+  push:
```

Comment:
I think we need to run it for every PR (on -> pull_request). For example, it seems that new checks haven't been run for this PR.